tests/utils/test_raster_embedding.py

```diff
@@ -0,0 +1,91 @@
+# test_raster_embedding.py
+
+import unittest
+import networkx as nx
+import numpy as np
+import dwave_networkx as dnx
+from minorminer.utils.raster_embedding import raster_embedding_search
+
+
+class TestRasterEmbedding(unittest.TestCase):
+
+    def testMinimalRaster(self):
+        for topology in ['chimera', 'pegasus', 'zephyr']:
+            if topology == 'chimera':
+                min_raster_scale = 1
+                S = dnx.chimera_graph(min_raster_scale)
+                T = dnx.chimera_graph(min_raster_scale+1)
+                num_emb = 4
+            elif topology == 'pegasus':
+                min_raster_scale = 2
+                S = dnx.pegasus_graph(min_raster_scale)
+                T = dnx.pegasus_graph(min_raster_scale+1)
+                num_emb = 2
+            elif topology == 'zephyr':
+                min_raster_scale = 1
+                S = dnx.zephyr_graph(min_raster_scale)
+                T = dnx.zephyr_graph(min_raster_scale+1)
+                num_emb = 2
+            embs = raster_embedding_search(S, T,
+                                           raster_breadth=min_raster_scale)
+            self.assertEqual(len(embs), num_emb,
+                             'mismatched number of embeddings')
+            self.assertTrue(all(set(emb.keys()) == set(S.nodes())
```

Comment:
use [assertSetEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertSetEqual) to be more specific